csp-guru/datapackage.json

```diff
@@ -92,128 +92,98 @@
-                        "type": "number"
-                    },
-                    {
-                        "name": "Remuneration_[USD/kWh;non_deflated]",
-                        "description": "indicates the level of FIT or PPA in [USD/kWh], non-deflated",
-                        "type": "number"
-                    },
-                    {
-                        "name": "Remuneration_[USD2017/kWh;deflated]",
-                        "description": "indicates the level of FIT or PPA in [USD2017/kWh], deflated from Year_operational using the Worldbank's GDP deflator; if station under development or constrution then deflated from 2017",
-                        "type": "number"
-                    },
-                    {
-                        "name": "Remuneration_[Euro/kWh;non_deflated]",
-                        "description": "indicates the level of FIT or PPA in [Euro/kWh], non-deflated",
-                        "type": "number"
-                    },
-                    {
-                        "name": "Remuneration_[Euro2017/kWh;deflated]",
-                        "description": "indicates the level of FIT or PPA in [Euro2017/kWh], deflated from Year_operational using the Worldbank's GDP deflator; if station under development or constrution then deflated from 2017 ",
-                        "type": "number"
-                    },
-                    {
-                        "name": "LCOE_5_25_[USD2017/kWh]",
-                        "description": "levelised cost of electricity with 5% weighted average cost of capital [USD2017/kWh] and a 25 year payback period, cap dependent O&M, deflated from Year_operational using the Worldbank's GDP deflator; if station under development or constrution then deflated from 2017",
-                        "type": "number"
-                    },
-                    {
-                        "name": "LCOE_10_25_[USD2017/kWh]",
-                        "description": "levelised cost of electricity with 10% weighted average cost of capital [USD2017/kWh] and a 25 year payback period, cap dependent O&M, deflated from Year_operational using the Worldbank's GDP deflator; if station under development or constrution then deflated from 2017",
-                        "type": "number"
+                        "name": "EPC_country",
+                        "description": "company responsible for engineering, procurement and construction of the CSP plant",
+                        "type": "string"
                     },
                     {
-                        "name": "LCOE_5_25_[Euro2017/kWh]",
-                        "description": "levelised cost of electricity with 5% weighted average cost of capital [Euro2017/kWh] and a 25 year payback period, deflated from Year_operational using the Worldbank's GDP deflator; if station under development or constrution then deflated from 2017",
-                        "type": "number"
+                        "name": "Receiver_Manufacturer",
```

Comment:
Is there any logic in your capitals? All other fields start with a capital letter and then they are all low caps. Would propose to do the same here.

Reply:
done.